public/app/features/admin/UserSyncInfo.tsx

```diff
@@ -45,7 +45,7 @@ export class UserSyncInfo extends PureComponent<Props, State> {
       <>
         <button className={`btn btn-secondary pull-right`} onClick={this.onSyncClick} disabled={isDisabled}>
           <span className="btn-title">Sync user</span>
-          {isSyncing && <Icon name="fa fa-spinner" className="fa-fw fa-spin run-icon" />}
```

Comment:
This `run-icon` className was repeating, but I couldn't find it implemented in any stylesheet, so I removed it.